### ISSUE TITLE
Network of YOLO Nano

### ISSUE THREAD

**Issue by ardeal** [AlexeyAB/darknet#5369]
Hi,

I am trying to train yolo nano.  
The publicized code is at: https://github.com/ardeal/yolo_nano

some information about my training:
+ RTX 2080Ti GPU + Pytorch + COCO
+ after 16 epochs training, the accuracy is still 0.0003

I tested pth model:  
load pth file and evaluate the model. For some images, the result is close to the ground truth, but not good. For some images, the result is entirely wrong.

I am really eager to know where the issue is.   
Could you please help to check my network and parameters and point out the issue?  
or could you please share you yolo nano network file?  

Thanks,  
Ardeal

**Comment by marcoslucianops** [comment]
[#4596](https://github.com/AlexeyAB/darknet/issues/4596#issuecomment-569753250)

**Comment by ardeal** [comment]
Hi,

What I really care about is not whether the accuracy is better or worse than yolo-tiny. 
what I really care about are:
1) reproduce the accuracy of YoloNano
2) what is wrong in my code. I would like to know whether the network in my code is correct or nor, and whether the parameters are correct or no.

Could you please help to check my network and parameters and point out the issue?
or could you please share you yolo nano network file?

Thanks,
Ardeal
